backend/daemon/api/entities/v1alpha/entities_test.go

```diff
@@ -170,3 +176,75 @@ func TestEntityTimeline(t *testing.T) {
 
 	testutil.ProtoEqual(t, want, timeline, "timeline must match")
 }
+
+func TestDeleteEntity(t *testing.T) {
+	//TODO(juligasa): this unit test is not really unitary since it depends on documents API.
```

Comment:
@juligasa Maybe this test should live in the `daemon_e2e_test.go` file?

Reply:
But there we have a much extensive test that covers this entirely wich is `TestAPIDeleteAndRestoreEntity` maybe we should remove `TestDeleteEntity` test then